capa/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2024 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+from logging import Formatter
+
+from rich.logging import RichHandler
+
+# use [/] after the logger name to reset any styling,
+# and prevent the color from carrying over to the message
+LOGFORMAT = "[grey54]%(name)s[/]: %(message)s"
+
+logging.basicConfig(level=logging.NOTSET)
```

Comment:
Please don't configure logging within a library, because the configuration will stomp over what an application may request. For capa, we configure logging within `capa.main`.

Reply:
logging now implemented in `capa.main`